titlecase/__init__.py

```diff
@@ -12,6 +12,9 @@
 import argparse
 import re
 import sys
+import os
+import pathlib
+import logging
```

Comment:
```suggestion
import argparse
import logging
import os
import pathlib
import re
import sys
```

Reply:
I'm not sure where I picked up the habit, but I usually try to alpha-order imports